src/SITES.html

```diff
@@ -22,7 +22,11 @@ <h1>
     CDN that performs the same job, but transparently.  There is no longer a
     list of registered mirrors, you can just point at <tt>www.cpan.org</tt> and go.
 </p>
-
+<p>To reconfigure the cpan client to remove old mirrors:</p>
```

Comment:
Can you rephrase this to be more about "reset to the default" ?

This isn't just about removing all mirrors but it's more about removing _all_ mirrors, and switching to the CDN solution instead.